nftserv/config.go

```diff
@@ -8,21 +8,29 @@ import (
 	"os"
 )
 
-type Config struct {
-	Host       string       `json:"host"`
-	Port       uint16       `json:"port"`
-	AssetsPath string       `json:"assetsPath"`
-	AssetsExt  string       `json:"assetsExt"`
-	CertFile   string       `json:"certFile"`
-	KeyFile    string       `json:"keyFile"`
-	Extra      ServerExtras `json:"extra"`
-}
+const defaultWhitelistedOrigin = "*"
 
-type ServerExtras struct {
-	WhitelistedOrigin    *string `json:"whitelistedOrigin"`
-	MaxTitleLength       *int    `json:"maxTitleLength"`
-	MaxDescriptionLength *int    `json:"maxDescriptionLength"`
-}
+type (
+	Config struct {
+		AssetsConfig AssetsConfig `json:"assetsConfig"`
+		ServerConfig ServerConfig `json:"serverConfig"`
```

Comment:
Since the containing type is already called `Config`, I propose
```suggestion
		Assets AssetsConfig `json:"assets"`
		Server ServerConfig `json:"server"`
```
also needs to be changed in demo json files.